mitodl/open-discussions
# incorporate course-search-utils into the CourseSearchPage

#### Pre-Flight checklist


#### What are the relevant tickets?

none

#### What's this PR do?

this PR adds the `@mitodl/course-search-utils` package and uses it on the CourseSearchPage, similarly to how it's used in hugo-course-publisher.

#### How should this be manually tested?

Thoroughly exercise *all* of the features on the course search page. Try to find issues where something might be not as it is on master - everything *should* be working 100% as on master.
